app/services/news_services/tweet_trending_news_item.rb

```diff
@@ -0,0 +1,173 @@
+module NewsServices
+  class TweetTrendingNewsItem < Patterns::Service
+    include Rails.application.routes.url_helpers
+    LOG_PREFIX = '[NewsTweet]'
+    MAX_ACCEPTABLE_REPLICATION_LAG = ::ApplicationHelper::MAX_ACCEPTABLE_REPLICATION_LAG
+
+    # Test run will not send to Twitter or save to database, but will still notify on slack
+    def initialize(min_tweet_interval: 4.hours, max_tweets_per_day: 4, min_votes_required: 10, leading_text: "", max_coin_tag_count: 3, test_run: false, store_test_run: false)
+      @min_tweet_interval = min_tweet_interval
+      @max_tweets_per_day = max_tweets_per_day
+      @min_votes_required = min_votes_required
+      @leading_text = leading_text
+      @max_coin_tag_count = max_coin_tag_count
+      @test_run = test_run
+      @store_test_run = store_test_run
+
+      @slack_channel = '#' + ENV.fetch('SLACK_CHANNEL_TRENDING_NEWS').gsub('#', '')
```

Comment:
I'm probably missing something but what's the purpose for prepending `#` and then `gsub`ing it back out?

Might be helpful to leave a code comment!

Reply:
Pretty sure we can't use hash tags with dot-env (it'll just end up as a comment in the .env file), but it'll probably work with env vars on production. Just wanted to not have to worry about whether or not to hashtag channel names.